content/shapes/functions.ts

```diff
@@ -2207,29 +3192,29 @@ function initLineup(slices: number, center: Point, radius: number, colors: {aCol
     }
     all.push(slice);
     slice.draw();
-  });
+  }
   return {all, groupA, groupB};
 }
 
 class Slice {
 
   private sides: {
-    path: GeoPath | null,
+    path: GeoPath | undefined,
     poly: Polyline
   }
   private arc: {
-    path: GeoPath | null,
+    path: GeoPath | undefined,
     poly: Arc
   }
   private _bounds: Polygon
   readonly width: number;
   private _tip: Point
   private _pivot: Point
   private _angle: number
-  private transAnim: AnimationResponse | null
-  private rotAnim: AnimationResponse | null
-  private rotTarget: number | null;
-  private transTarget: Point | null;
+  private transAnim: AnimationResponse | undefined
+  private rotAnim: AnimationResponse | undefined
+  private rotTarget: number | undefined;
+  private transTarget: Point | undefined;
```

Comment:
Rather than explicitly listing `undefined` as an option, you can do `private transTarget?: Point` with a question mark.